### ISSUE TITLE
WCAG: Predictable

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/hourglass#341]
# Summary

This review has considered:
1. On Focus
2. On Input
3. Consistent Navigation
4. Consistent Identification
5. Consistent Help

- #370
- https://github.com/BirkbeckCTP/janeway/issues/4138
- [ ] add to style guidelines for future updates that any content which about contacting a member of the team should refer direct to the contact us page rather than include the email address directly in that content.

# Audit Results

## On Focus
There are no scripts to remove focus when it is received.

None of the links open in a new window.

## On Input
There are submit buttons for all the forms.

Filters activate as soon as terms are typed, but this will be an anticipated and expected change as part of the function of a filter.

The error message following an attempt to log in with an incorrect username & password does fail this criterion.  This has also been discussed in terms of 2.2 Enough time in #333  and has been added as an action point there.

## Consistent Navigation
There is a standard Nav Bar.  However it does differ in order between the wide and narrow screen versions, as already discussed under #338 

## Consistent Identification

The styling of the pages is consistent, with reusable components.

## Consistent Help
The only help available is through a 'contact us' form on a specific page, and as part of the proof-reading of the site we were noting down when any contact details were presented outside of directing to that page.  All contact is being directed through that page.

**Comment by joemull** [comment]
I've added a few issues to address this.

**Comment by StephDriver** [comment]
Discussion JM: SD add consistency with contact us to branding A11y appendix, then it can close.
